server/service/osquery_utils/queries.go

```diff
@@ -2213,15 +2213,18 @@ func directIngestHostCertificates(
 	for _, row := range rows {
 		csum, err := hex.DecodeString(row["sha1"])
 		if err != nil {
-			return ctxerr.Wrap(ctx, err, "directIngestHostCertificates: decoding sha1")
+			level.Error(logger).Log("component", "service", "method", "directIngestHostCertificates", "msg", "decoding sha1", "err", err)
+			continue
 		}
 		subject, err := fleet.ExtractDetailsFromOsqueryDistinguishedName(row["subject"])
 		if err != nil {
-			return ctxerr.Wrap(ctx, err, "directIngestHostCertificates: extracting subject details")
+			level.Error(logger).Log("component", "service", "method", "directIngestHostCertificates", "msg", "extracting subject details", "err", err)
+			continue
 		}
 		issuer, err := fleet.ExtractDetailsFromOsqueryDistinguishedName(row["issuer"])
 		if err != nil {
-			return ctxerr.Wrap(ctx, err, "directIngestHostCertificates: extracting issuer details")
+			level.Error(logger).Log("component", "service", "method", "directIngestHostCertificates", "msg", "extracting issuer details", "err", err)
+			continue
```

Comment:
As a follow up product question, we should determine what we want to store for a certificate (if anything) where we can't extract the subject or issuer info.